directives/tooltip/tooltip_directive_default.story.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <dt-stack
+    direction="row"
+    gap="600"
+    class="d-mt64"
+  >
+    <dt-button
+      v-dt-tooltip:top-end="'Tooltip text on top-end'"
+      importance="clear"
+      kind="danger"
+    >
+      Trying complex examples
```

Comment:
what does "trying complex examples" mean exactly?

Reply:
That was a leftover, as I was copying the vnodes, I was trying a "complex" example of having children, properties and several stuff but now that's not important as I'm not longer cloning vnodes.

Will change wording there.